OADP-2146: release notes for OADP 1.2.1 (#62921)
openshift/openshift-docs: modules/oadp-release-notes-1-2-1.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/oadp-release-notes.adoc

:_content-type: REFERENCE
[id="migration-oadp-release-notes-1-2-1_{context}"]
= OADP 1.2.1 release notes

[id="new-features-1-2-1_{context}"]
== New features

There are no new features in the release of {oadp-first} 1.2.1.

// :FeatureName: OADP Data Mover
// include::snippets/technology-preview.adoc[]

[id="resolved-issues-1-2-1_{context}"]
== Resolved issues

For a complete list of all issues resolved in the release of OADP 1.2.1, see the list of link:https://issues.redhat.com/issues/?filter=12417849[OADP 1.2.1 resolved issues] in Jira.

// Jira filter - project = OADP AND issuetype = Bug AND status in (Verified, "Release Pending", Closed) AND priority in (Blocker, Critical, Major) AND fixVersion = "OADP 1.2.1" AND component != Documentation

[id="known-issues-1-2-1_{context}"]
== Known issues

The following issues have been highlighted as known issues in the release of OADP 1.2.1:

.DataMover Restic retain and prune policies do not work as expected

> The retention and prune features provided by VolSync and Restic are not working as expected. Since there is no working option to set the prune interval on VolSync replication, customers will have to manage and prune remotely stored backups on S3 storage outside of OADP. For more details, see:

Reviewer: Since - To facilitate translation, use this word only to refer to time. Do not use it as a synonym for because.

```suggestion
The retention and prune features provided by VolSync and Restic are not working as expected. Because there is no working option to set the prune interval on VolSync replication, you have to manage and prune remotely stored backups on S3 storage outside of OADP. For more details, see:
```
